src/main/java/com/microsoft/sqlserver/jdbc/Util.java

```diff
@@ -373,9 +373,14 @@ else if (ch == ':')
                         name = SQLServerDriver.getNormalizedPropertyName(name, logger);
                         if (null != name) {
                             if (logger.isLoggable(Level.FINE)) {
-                                if ((false == name.equals(SQLServerDriverStringProperty.USER.toString()))
-                                        && (false == name.equals(SQLServerDriverStringProperty.PASSWORD.toString())))
```

Comment:
just for my own curiosity, why removed `&& (false == name.equals(SQLServerDriverStringProperty.PASSWORD.toString()))` ?